src/render/Renderer.java

```diff
@@ -152,14 +150,14 @@ private void renderWorld(float lerp, World world) {
         spriteBatch.render();
 
         spriteBatch.setTexture(terrainTexture);
+        int texY = (HEX_HEIGHT + 1) * STYLE;
```

Comment:
Could you create variable SPRITE_PADDING (or better name) and use that in place of the magic 1 here?
Also change texX = 15, 30, 45, etc. to use HEX_WIDTH and SPRITE_PADDING .  (Ex. column * (HEX_WIDTH + SPRITE_PADDING )).

Reply:
It's fairly normal to use magic numbers for sprite locations, or even read them from a file. They're not guaranteed to be regularly spaced unless made with that requirement in mind. But sure, since we do have a regularly spaced one, I updated it.